gtubt/lib/ui/components/hamburger_menu.dart

```diff
@@ -43,35 +48,87 @@ class _HamburgerMenuComponentsState extends State<HamburgerMenuComponents> {
     );
   }
 
+  void _settingsButtonFunction() {
+    BlocProvider.of<PageBloc>(context).add(
+      PageChanged(
+        context: context,
+        routeName: SETTINGS_URL,
+      ),
+    );
+  }
+
+  Widget _buildProfileHeader(String username, String profileImage) {
+    return Column(
+      children: [
+        Container(
+            width: 150,
+            height: 150,
+            decoration: BoxDecoration(shape: BoxShape.circle),
+            clipBehavior: Clip.antiAlias,
+            child: FittedBox(
+              child: profileImage != null
```

Comment:
ternary operatorleri kullanmak guzel amma bunu bir metoda alalim. kodda okunabilirligi dusuruyor bu flutter'in yapisindan kaynakli.